Content.Shared/Physics/Controllers/SharedConveyorController.cs

```diff
@@ -0,0 +1,169 @@
+﻿using Content.Shared.Conveyor;
+using Content.Shared.Gravity;
+using Content.Shared.Movement.Systems;
+using Robust.Shared.Map;
+using Robust.Shared.Physics;
+using Robust.Shared.Physics.Components;
+using Robust.Shared.Physics.Controllers;
+using Robust.Shared.Physics.Events;
+using Robust.Shared.Physics.Systems;
+
+namespace Content.Shared.Physics.Controllers;
+
+public abstract class SharedConveyorController : VirtualController
+{
```

Comment:
Because this is abstract client is never going to run this code, only server which inherits this.